app/src/main/java/couclou/fr/nixconfighelper/MainAdapter.kt

```diff
@@ -25,24 +39,126 @@ class MainAdapter(val feed:Feed): RecyclerView.Adapter<CustomViewHolder>() {
+                                        "Github : ${element.maintainers[0].github}\n" +
+                                        "Github id : ${element.maintainers[0].githubId}"
+                    }
+                    else -> {
+                        for (iterator in element.maintainers.indices) {
+                            when (iterator) {
+                                0 -> {
+                                    maintainers = "Name : ${element.maintainers[iterator].name}\n" +
+                                                    "Email : ${element.maintainers[iterator].email}\n" +
+                                                    "Github : ${element.maintainers[iterator].github}\n" +
+                                                    "Github id : ${element.maintainers[iterator].githubId}\n"
+                                }
+                                element.maintainers.size - 1 -> {
+                                    maintainers = "$maintainers\nName : ${element.maintainers[iterator].name}\n" +
+                                                    "Email : ${element.maintainers[iterator].email}\n" +
+                                                    "Github : ${element.maintainers[iterator].github}\n" +
+                                                    "Github id : ${element.maintainers[iterator].githubId}"
+                                }
+                                else -> {
+                                    maintainers = "$maintainers\nName : ${element.maintainers[iterator].name}\n" +
+                                                    "Email : ${element.maintainers[iterator].email}\n" +
+                                                    "Github : ${element.maintainers[iterator].github}\n" +
+                                                    "Github id : ${element.maintainers[iterator].githubId}\n"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            var outputsToInstall: String = ""
+            if (element.outputsToInstall.isNotEmpty()) {
+                when (element.outputsToInstall.size) {
+                    1 -> outputsToInstall = element.outputsToInstall[0]
+                    else -> {
+                        for (iterator in element.outputsToInstall.indices) {
+                            when (iterator) {
+                                0 -> outputsToInstall = element.outputsToInstall[iterator]
+                                else -> outputsToInstall = "$outputsToInstall\n${element.outputsToInstall[iterator]}"
+                            }
+                        }
+                    }
+                }
+            }
+
+            var platforms: String = ""
+            if (element.platforms.isNotEmpty()) {
+                for (platform in element.platforms) {
+                    platforms = "$platforms \n $platform"
```

Comment:
Cf https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/join-to-string.html